src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    BetDao betDao;
+    @Inject
+    UserDao userDao;
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            User user = null;
+            try {
+                String[] data = command.split(" ");
+                String name = data[0];
+                int age = Integer.parseInt(data[1]);
+                int value = Integer.parseInt(data[2]);
+                double risk = Double.parseDouble(data[3]);
+                user = new User(name, age);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException | IndexOutOfBoundsException e) {
+                System.out.println("Please, input correct data");
+            }
+            betDao.add(bet);
+            userDao.add(user);
+            System.out.println(bet == null ? null : user.toString() + " " + bet.toString());
```

Comment:
There is no need to output the null bet. You should check the input data only on its validity. If it's invalid - output the message for the user to check the input data or typing in "q" for quitting the service.

Reply:
done